bower.json

```diff
@@ -15,5 +15,8 @@
   "ignore": [
     "**/.*",
     "!dist/braintree-angular.js"
-  ]
+  ],
+  "dependencies": {
+    "braintree-web": "2.5.3"
+  }
```

Comment:
From what I understand about bower I don't think this is necessary. `dist/braintree-angular.js` bundles a copy of braintree-web with it.